Northstar.CustomServers/mod/scripts/vscripts/lobby/sh_lobby.gnut

```diff
@@ -16,8 +16,8 @@ global struct CustomMatchSettingContainer
 
 struct {
 	array<string> modes = [ // default modes in vanilla
-		"tdm",
 		"aitdm",
+		"tdm",
```

Comment:
... really, why?

Reply:
(for context, you removed this from above and put it again below, check the full diff)

Reply:
Had to move this around for vanilla modes ui for some odd reason